matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/timeline/TimelineService.kt

```diff
@@ -55,4 +55,40 @@ interface TimelineService {
      * Returns a snapshot list of TimelineEvent with EventType.MESSAGE and MessageType.MSGTYPE_IMAGE or MessageType.MSGTYPE_VIDEO.
      */
     fun getAttachmentMessages(): List<TimelineEvent>
+
+    /**
+     * Get a live list of all the thread for the specified roomId
+     * @return the [LiveData] of [TimelineEvent]
+     */
+    fun getAllThreadsLive(): LiveData<List<TimelineEvent>>
```

Comment:
Yes, maybe it would be good, however some of the functions are for the main timeline like:
`getNumberOfLocalThreadNotificationsLive`
`getNumberOfLocalThreadNotifications`

Reply:
Ok, so splitting into 2 different interfaces will definitely make it clearer!

Reply:
Done